.github/workflows/tests.yml

```diff
@@ -13,6 +13,8 @@ on:
   push:
     branches:
       - master
+      - "feature/*"
```

Comment:
I would also add it on `pull_request`, since we sometimes develop and open PRs against the `features/*`